comment/views.py

```
@@ -0,0 +1,173 @@
+
+class CommentDetailView(APIView):
+  @swagger_auto_schema(
+    operation_id="댓글 삭제",
+    operation_description="댓글을 삭제합니다.",
+    request_body=CommentRequestSerializer,
+    responses={204: "No Content", 400: "Bad Request", 401: "Unauthorized", 404: "Not Found"},
+    )
+  def delete(self, request, comment_id):
+    try: 
+      comment = Comment.objects.get(id=comment_id)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)